lib/transaction.js

```diff
@@ -2,7 +2,7 @@
 import { AccountID, Address } from './account.js';
 import { None, Some } from './prelude.js';
 import { SubmitResult } from './schema.js';
-import { base58ToBytes, base58ToHex, bytesToHex, intToHex } from './utils.js';
+import { base58ToBytes, base58ToHex, bytesToHex, hexToBytes, intToHex, } from './utils.js';
```

Comment:
do we need the extra `,` after `intToHex` @0x3bfc ?

Reply:
This was added by the prettifier